Makefile

```diff
@@ -236,13 +237,18 @@ test: $(DOCKER_BUILDER)
 	$(call docker_builder_make,$@)
 endif
 
+hello-profiler: build
+$(OUT_BIN_HELPRO): go/deps
+	$(GO_ENV) $(GO) build $(SANITIZERS) $(GO_BUILD_FLAGS) -o $@ ./cmd/hello-profiler
+
+
 cputest-static: build
 	$(GO_ENV) $(CGO_ENV) $(GO) test -v ./pkg/profiler/cpu -c $(GO_BUILD_FLAGS) --ldflags="$(CGO_EXTLDFLAGS)"
-	mv cpu.test kerneltest/
+	mv cpu.test kerneltest/arm64/
 
 initramfs: cputest-static
-	bluebox -e kerneltest/cpu.test
-	mv initramfs.cpio kerneltest
+	bluebox -e kerneltest/cpu.test -o amd64-initramfs.cpio
```

Comment:
Do we even still need this make target with all the instructions in the github action for now? I would prefer if we have one source of truth, either we have a script that we call from the github action, make target(s), or everything in the github action. It's a bit all over the place right now.

Reply:
I agree with you. The targets are largely still helpful for local dev.

The reason we have things slightly different in github actions is that CI and goreleaser do some weird things (like make binaries non-executable in this case). 

We ultimately do end up calling the `vmtest.sh` script in both cases, once we are past the goreleaser part. I'll still see if we can deduplicate some more. 

Since Kemal's planning to reimplement the whole thing I just wanted to hold off.

Reply:
Nuking the make targets for now, will try to write a script in a follow up PR that we can run both locally and in CI

Reply:
sounds good